pkg/plugins/manager/pipeline/bootstrap/steps.go

```diff
@@ -86,6 +87,22 @@ func AppDefaultNavURLDecorateFunc(_ context.Context, p *plugins.Plugin) (*plugin
 	return p, nil
 }
 
+// TemplateDecorateFunc is a DecorateFunc that removes the placeholder for the version and last_update fields.
+func TemplateDecorateFunc(_ context.Context, p *plugins.Plugin) (*plugins.Plugin, error) {
+	// %VERSION% and %TODAY% are valid values, according to the plugin schema
+	// but it's meant to be replaced by the build system with the actual version and date.
+	// If not, it's the same than not having a version or a date.
+	if p.Info.Version == "%VERSION%" {
+		p.Info.Version = ""
```

Comment:
Wouldn't it make sense to set the Grafana version, e.g. 9.2.0-pre, here rather than leaving it empty. Same thing with the updated field "now".  Maybe I'm moving ahead of time, but we want to start storing the plugin version in dashboards eventually - I think we definitely should create an issue to track this if we don't have one.

Should this apply to any plugin or just core ones?

Reply:
I think this needs a bit more of discussion:

- For core plugins, if we just do that (`%VERSION%` --> Grafana version) we would have an inconsistency between different core plugins. Some will show a version and some others don't. I think that if we want to do that we would need to also substitute empty versions for the Grafana version. That's out of the scope of what this PR was meant to fix but I could add that if you think so.
- For external plugins, we should keep it as it is in this PR because the Grafana version is not relevant.

Reply:
These changes should be fine, my main point was "I think we definitely should create an issue to track this if we don't have one" and/or start a design doc

Reply:
Sure will do :+1:

Reply:
https://github.com/grafana/grafana/issues/76419